_config.yml

```diff
@@ -4,11 +4,12 @@ name: Blog Name
 description: Software Developer
 
 # Url is currently only used only for the RSS feed in feed.xml
-url: http://streetturtle.github.io
+url: ''
+#url: http://streetturtle.github.io
 #url: http://localhost:4000
 
 # For a project page here should be a project name. Otherwise (for user's/organizaion's page leave it empty)
-baseurl: '/jekyll-clean-dark'
+baseurl: ''
```

Comment:
These two parameters are needed to make this theme be accessible by [streetturtle.github.io/jekyll-clean-dark](streetturtle.github.io/jekyll-clean-dark).